### ISSUE TITLE
STY: Move to pydata-sphinx-theme

### ISSUE THREAD

**Issue by larsoner** [mne-tools/mne-study-template#264]
Now that NumPy, SciPy (in progress), pandas, and (most importantly I think) MNE-Python have moved to pydata-sphinx-theme, it might be nice to move mne-study-template over, too, for cohesiveness. Any thoughts or objections?

**Comment by agramfort** [comment]
-1 here.

we use mkdocs not sphinx

>

**Comment by larsoner** [comment]
Why not move to sphinx, too, then?

**Comment by larsoner** [comment]
(To me this would reduce overall maintenance burden since then all of these MNE projects can share the same tips/tricks/knowledge/workflow, etc.)

**Comment by agramfort** [comment]
We had this conversation with @hoechenberger and he made it work quickly
and it's nice so I agreed

>

**Comment by larsoner** [comment]
I hear you saying that if I convince @hoechenberger you'll be on board :)

**Comment by hoechenberger** [comment]
You can try but you will fail :) no way I'll voluntarily move to Sphinx after tasting the sweet syrup of mkdocs-material ☺️ Thr search is much nicer; it comes with a file watcher to auto-regenerate and auto-reload the docs in your browser; it uses markdown, which I prefer over RST; and configuration is much simpler

So yeah... you can try but ... 😅

**Comment by hoechenberger** [comment]
Whoops, closed without a further comment. I hope I didn't come across as too rude! I appreciate your offer / idea to consolidate themes across all MNE projects. It's just that I really like Mkdocs in looks, usability, and developability (?!) Feels like modern web development to me. In contrast, Sphinx may be much more powerful, but it feels sooo pre-Web-2.0.

**Comment by larsoner** [comment]
Sorry, I should have said "okay!" or something similar. I figured at the end of the day we're going to have to agree to disagree about this because we're just prioritizing things differently, and that's okay. But since you asked... :)

To me the (by far) most important consideration here is consistency of the MNE user experience (and nowadays, their experience in the broader scientific Python ecosystem), and everything else should take a back seat to that.

The second most important thing to me is consistency of the dev experience so that we aren't maintaining `N` different ways of doing the same thing. For example, I maintain at least 6 packages that all use the same CircleCI build infrastructure, so when one breaks in some way, I figure out how to fix that one, and just apply the same fix 6 times. If I used different tools in each repo, I'd have to do 6 times the amount of work to keep things running.

So in the end I totally agree Sphinx and RST are painful to work with, but I'd rather move all `MNE-*` projects over to something else at the same time if there is a better tool, and only if (nowadays) it allowed us to stay consistent with the rest of the scientific python ecosystem (giving the same experience as `pydata-sphinx-theme`).
